src/PlusDataCollection/Atracsys/AtracsysTracker.cxx

```diff
@@ -793,32 +857,32 @@ AtracsysTracker::ATRACSYS_RESULT AtracsysTracker::GetMarkersInFrame(std::vector<
   return SUCCESS;
 }
 
-//----------------------------------------------------------------------------
-AtracsysTracker::ATRACSYS_RESULT AtracsysTracker::EnableIRStrobe(bool enabled)
-{
-  if (ftkSetInt32(this->Internal->FtkLib, this->Internal->TrackerSN, OPTION_IR_STROBE, enabled) != ftkError::FTK_OK)
-  {
-    return ERROR_ENABLE_IR_STROBE;
-  }
-  return SUCCESS;
-}
-
 //----------------------------------------------------------------------------
 AtracsysTracker::ATRACSYS_RESULT AtracsysTracker::SetUserLEDState(int red, int green, int blue, int frequency, bool enabled /* = true */)
 {
-  if (ftkSetInt32(this->Internal->FtkLib, this->Internal->TrackerSN, OPTION_LED_FREQUENCY, frequency) != ftkError::FTK_OK)
+  // get correct device option number
+  ftkOptionsInfo* info;
+  if (!GetOptionInfo("User-LED frequency", info))
```

Comment:
Are there constants somewhere in the API that can be referenced instead of writing the strings here?

Reply:
The only constants in the API are the option names themselves. As explained in the PR description, we may change the actual id of options in the future, not their names, hence this approach of using the strings instead of the hardcoded id.
What we could do though is define static constants for all these strings in the wrapper (e.g. where the `enum OPTIONS` used to be) in order to put them all in one place. Let me know if you think that's worth doing.

Reply:
I don't think that the previous option was much better. If there were option IDs or names exposed in the headers I would suggest using those instead, but there don't seem to be any.